src/service/auth/AuthService.ts

```diff
@@ -1,17 +1,21 @@
 import { HttpError, HttpStatusCode } from '../HttpError'
+import { findUserByEmail } from './AuthRequest'
 
 const jwt = require('jsonwebtoken')
 
-export const createAccessToken = (emailUser, passwordUser) => {
-  const { USERS, SECRET } = process.env
-  const listUsers = JSON.parse(USERS)
-  const emailFound = listUsers.filter(users => users.email == emailUser)
+export const createAccessToken = async (emailUser, passwordUser) => {
+  const { SECRET, NODEMAILER_SECRET } = process.env
 
-  if (emailFound.length === 0 || emailFound[0].password !== passwordUser) {
+  const encodePassword = jwt.sign(passwordUser, NODEMAILER_SECRET)
+
+  const user = await findUserByEmail(emailUser)
```

Comment:
O que acontece se nenhum usuário for encontrado com aquele email? Eu chuto que o repositório retornará `null` ou `undefined`, o que provavelmente quebrará o `if` na linha 14.